content/administration/maintain/supported_versions.rst

```diff
@@ -64,6 +66,9 @@ This matrix shows the support status of every version.
     🏁 Future version, not released yet
 
 
+.. This is an awful way to display colored circles but "Large Green Circle Emoji" (U+1F7E2) was only
+   released in 2019. Let's wait a few more years before switching to emojis.
```

Comment:
you can add a link to https://unicode-table.com/en/1F7E2/ so we can retrieve it easily